lib/ConnectionConfig.js

```diff
@@ -93,7 +97,7 @@ ConnectionConfig.getCharsetNumber = function getCharsetNumber(charset) {
   return num;
 };
 
-ConnectionConfig.getDefaultFlags = function getDefaultFlags(options) {
+ConnectionConfig.getDefaultFlags = function getDefaultFlags(multipleStatements) {
```

Comment:
This is not going to be backwards-compatible since users have been able to call this. Just leave it as `options`.